<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>ktelephone</class>
    <widget class="QMainWindow" name="ktelephone">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>307</width>
                <height>139</height>
            </rect>
        </property>
        <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
            </sizepolicy>
        </property>
        <property name="minimumSize">
            <size>
                <width>307</width>
                <height>139</height>
            </size>
        </property>
        <property name="maximumSize">
            <size>
                <width>307</width>
                <height>139</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>ktelephone</string>
        </property>
        <property name="dockNestingEnabled">
            <bool>true</bool>
        </property>
        <widget class="QWidget" name="centralWidget">
            <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                </sizepolicy>
            </property>
            <property name="maximumSize">
                <size>
                    <width>420</width>
                    <height>16777215</height>
                </size>
            </property>
            <layout class="QGridLayout" name="gridLayout">
                <item row="1" column="0">
                    <widget class="QLineEdit" name="sipInput"/>
                </item>
                <item row="0" column="0">
                    <widget class="QComboBox" name="statusComboBox">
                        <item>
                            <property name="text">
                                <string>Available</string>
                            </property>
                        </item>
                        <item>
                            <property name="text">
                                <string>Unavailable</string>
                            </property>
                        </item>
                    </widget>
                </item>
            </layout>
        </widget>
        <widget class="QMenuBar" name="menuBar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>307</width>
                    <height>26</height>
                </rect>
            </property>
            <widget class="QMenu" name="menuSettings">
                <property name="title">
                    <string>Setti&amp;ngs</string>
                </property>
                <addaction name="actionPreferences"/>
            </widget>
            <widget class="QMenu" name="menuHelp">
                <property name="title">
                    <string>He&amp;lp</string>
                </property>
                <addaction name="actionAbout"/>
            </widget>
            <addaction name="menuSettings"/>
            <addaction name="menuHelp"/>
        </widget>
        <widget class="QStatusBar" name="statusBar"/>
        <action name="actionPreferences">
            <property name="text">
                <string>&amp;Preferences...</string>
            </property>
        </action>
        <action name="actionAbout">
            <property name="text">
                <string>&amp;About...</string>
            </property>
        </action>
    </widget>
    <resources/>
    <connections/>
</ui>
